src/server/scripts/World/go_scripts.cpp

```diff
@@ -1212,7 +1212,55 @@ class go_bells : public GameObjectScript
         return new go_bellsAI(go);
     }
 };
+/*####
+## go_ETC70_music
+####*/
+
+enum ETC70Music {
+    POWER_OF_THE_HORDE = 181075,
+};
+enum ETC70MusicEvent
+{
+    EVENT_ETC_START_MUSIC = 1
+};
+class go_etc_concert_music : public GameObjectScript
+{
+public:
+    go_etc_concert_music() : GameObjectScript("go_etc_concert_music") { }
+
+    struct go_etc_concert_musicAI : public GameObjectAI
+    {
+        go_etc_concert_musicAI(GameObject* go) : GameObjectAI(go)
+        {
+            _events.ScheduleEvent(EVENT_ETC_START_MUSIC, 1s);
+        }
+
+        void UpdateAI(uint32 diff) override
+        {
+            _events.Update(diff);
+            while (uint32 eventId = _events.ExecuteEvent())
+            {
+                switch (eventId)
+                {
+                case EVENT_ETC_START_MUSIC:
 
+                    me->PlayDirectMusic(POWER_OF_THE_HORDE);
```

Comment:
```suggestion
                case EVENT_ETC_START_MUSIC:
                    me->PlayDirectMusic(POWER_OF_THE_HORDE);
```
unnecessary line break here